acsoo/addons.py

```diff
@@ -31,21 +41,10 @@ def _split_set(csv):
 def addons(ctx, addons_dirs, include, exclude, separator):
     include = _split_set(include)
     exclude = _split_set(exclude)
-    if not addons_dirs:
-        addons_dirs = []
-        candidate_addons_dirs = (
-            os.path.join('odoo', 'addons'),
-            'odoo_addons',
-            '.',
-        )
-        for addons_dir in candidate_addons_dirs:
-            if os.path.isdir(addons_dir):
-                addons_dirs.append(addons_dir)
     manifests = {}
-    for addons_dir in addons_dirs:
-        for addon, manifest in get_installable_addons(addons_dir).items():
-            if (not include or addon in include) and addon not in exclude:
-                manifests[addon] = manifest
+    for addon, manifest in get_installable_addons(addons_dirs).items():
+        if (not include or addon in include) and addon not in exclude:
+            manifests[addon] = manifest
```

Comment:
Actually, I think this refactoring was not necessary.

Reply:
Indeed as you see in `addons_list_depends` and `addons_list`, a subcommand of acsoo addons has access to the manifests through `ctx.obj['manifests']`. So `toupdate` can do the same. 

It just needs to obtain the manifest path. So `get_installable_addons` Could return two dictionaries, or a dict {addon_name: (manifest_path, manifest_dict)}.